projecttacoma/bulk-export-server
# EXM347 ignore

# Summary
After an investigation, the reference issues in EXM347 extended beyond the initial errors we reported. Instead of trying to fix the issues, we've decided to ignore the bundle from the connectathon upload script.

## New behavior
$import requests to the export server should work now

## Code changes
Added `EXM347-4.3.000-bundle.json` to the ignored bundles array in the connectathon upload script

# Testing guidance
- Run `npm run db:reset`
- Run `npm run connectathon-upload`
- Run `PORT=3001 npm run start`
- Open the test server and run `npm run start`
- Send an $import request to the bulk export server with the following request body: 
```
{
    "resourceType": "Parameters",
    "parameter": [
        {
            "name": "exportUrl",
            "valueString": "http://localhost:3001/$export"
        }
    ]
} 
```
- Query the bulkstatus endpoint and ensure the request succeeds!
